packages/telemetry/test/span-exporters/console-trace-link-exporter.spec.ts

```diff
@@ -0,0 +1,168 @@
+import { expect } from 'chai'
+
+import { ConsoleTraceLinkExporter } from '../../src/span-exporters/console-trace-link-exporter'
+
+describe('consoleTraceLinkExporter', () => {
+  describe('new', () => {
+    it('sets up trace url', () => {
+      const exporter = new ConsoleTraceLinkExporter({
+        serviceName: 'serviceName',
+        team: 'team',
+        environment: 'environment',
+      })
+
+      //@ts-ignore
+      expect(exporter._traceUrl).to.equal('https://ui.honeycomb.io/team/environments/environment/datasets/serviceName/trace?trace_id')
+    })
+  })
+
+  describe('export', () => {
+    it('logs the start of the first span with a unique trace', (done) => {
+      const exporter = new ConsoleTraceLinkExporter({
+        serviceName: 'serviceName',
+        team: 'team',
+        environment: 'environment',
+      })
+
+      //@ts-ignore
+      exporter._log = (...args) => {
+        expect(args[0]).to.equal('Trace start: [spanName] - https://ui.honeycomb.io/team/environments/environment/datasets/serviceName/trace?trace_id=traceId')
+      }
+
+      exporter.export([{
+        name: 'spanName',
+        //@ts-ignore
+        spanContext: () => {
+          return {
+            traceId: 'traceId',
+            spanId: 'spanId',
+          }
+        },
+      }], (result) => {
+        //@ts-ignore
```

Comment:
```suggestion
        //@ts-expect-error
```

Reply:
probs makes sense for most of these to be `ts-expect-error` but it isn't really that big of a deal in a test file 🤷🏻‍♂️